### PR TITLE
Iris Readme updated for rev3: dfu and reference to udev rules

### PR DESCRIPTION
**Pull request by jkurthoconnor**
## Description

Add a second flashing `make` command for rev3 with dfu
Add link to docs' `udev` rules for linux users

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [X ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/iris/readme.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ Example of flashing this keyboard:
 
     make keebio/iris/rev2:default:avrdude
 
-See [build environment setup](https://docs.qmk.fm/#/newbs_getting_started) then the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information.
+Note that Iris rev3 uses the `dfu` bootloader:
+
+    make keebio/iris/rev3:default:dfu
+
+See [build environment setup](https://docs.qmk.fm/#/newbs_getting_started) then the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Linux users should consult options for [`atmel-dfu` `udev` rules](https://docs.qmk.fm/#/faq_build?id=linux-udev-rules).
```
Comment (drashna) [inline keyboards/keebio/iris/readme.md]: While the first part is fine, I am not sure that this should be here.  Specifically, this info is not board specific, and applies to any board.  As such, I feel that it's outside of the scope for what this readme is supposed to be.

```suggestion
```

Comment (noroadsleft) [inline keyboards/keebio/iris/readme.md]:
```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

**Comment by drashna** [comment]
Any updates on this?

**Comment by zvecr** [comment]
My preference would be to wait for #6224

**Comment by yanfali** [comment]
Fair, but tbh, this would still continue to work even when we add new capabilities.

**Comment by drashna** [comment]
maybe we should update a bunch of the readmes at once, after #6224 gets merged?

**Comment by drashna** [comment]
Well, #6224 has been merged.

**Pull request closed without merging**